runtime/src/tasks/fifo/tasks-fifo.c

```diff
@@ -993,3 +997,16 @@ uint32_t chpl_task_getNumThreads(void) {
 uint32_t chpl_task_getNumIdleThreads(void) {
   return idle_thread_cnt;
 }
+
+// Misc
+
+static void warn_threads_per_locale(void) {
+  if (getenv("CHPL_RT_NUM_THREADS_PER_LOCALE_QUIET")) return;
```

Comment:
We have `chpl_env_rt_get_bool` wrappers I think'd we'd want to use (it does truthy/falsey checks so you could have `CHPL_RT_NUM_THREADS_PER_LOCALE_QUIET=false`). 

Not something you have to do, but if you were copying code from `chpl_task_getenvNumThreadsPerLocale`, I believe most of that code predates the wrappers. It seems like that code could be cleaned up by using the wrappers though (won't be as straightforward as just using the int parser since we have sentinel values, but may clean things up to use `chpl_env_rt_get` and then `chpl_env_str_to_int` when we need the int.)

`chpl_task_getEnvCallStackSize` also seems like it could be simplified with `chpl_env_str_to_size`, but I don't mean to add to what was supposed to be a simple PR, just noting is since I found it while looking at the other.

Reply:
Looked at using the wrappers in `chpl_task_getenvNumThreadsPerLocale` and there's enough different in error handling that I'm leaving it unless there's a major push to clean it up.